src/main/java/com/cleanroommc/groovyscript/command/GSCommand.java

```diff
@@ -179,6 +179,10 @@ public GSCommand() {
                                        .appendSibling(GSCommand.getTextForFile("Groovy Log", GroovyLog.get().getLogFilerPath().toString(), new TextComponentString("Click to open GroovyScript log"))));
         }));
 
+        addSubcommand(new SimpleCommand("deleteClassCache", (server, sender, args) -> {
```

Comment:
"classes" are something distinct from other scripts in GroovyScript, but this clears all cached files. think this should be named `deleteCache`